updater/Gemfile.lock

```diff
@@ -122,7 +122,7 @@ GEM
       public_suffix (>= 2.0.2, < 6.0)
     ast (2.4.2)
     aws-eventstream (1.2.0)
-    aws-partitions (1.752.0)
+    aws-partitions (1.759.0)
```

Comment:
This feels weird / risky to me that we're getting bumps of this and `mime-types-data` in a version bump. 

IMO we should only be getting the `dependabot-*` bumps in a version release PR.

Reply:
I think it's related to `bundle lock` being run with an old Bundler version since I believe this was fixed by https://github.com/rubygems/rubygems/pull/6184, released with Bundler 2.4.0.

Reply:
Ah that makes sense, since this was still using `bundler` `2.3.26`... glad to hear it'll go away once we are using newer bundler...

IMO even if we go with https://github.com/dependabot/dependabot-core/pull/7231 and have an older version of bundler for a bit still pulling these in, it'll be fine for both the reasons I mentioned ☝️ and also because we'll soon enough be on a newer ruby/bundler version...

Reply:
This will be fixed by https://github.com/dependabot/dependabot-core/pull/7239